src/redux/reducers/settings.js

```diff
@@ -65,13 +65,16 @@ export default handleActions(
         avatarType,
       } = settings;
       let { customThemeValue } = settings;
-      const themeArray = customThemeValue.split(',').map(value => `#${value}`);
+      let settingsTheme = theme ? theme : defaultState.theme;
```

Comment:
Directly write `theme = defaultState.theme` while destructuring settings.

Reply:
Okay, on it